src/components/GroupedTaskStatusBadge/index.tsx

```diff
@@ -1,29 +1,62 @@
 import styled from "@emotion/styled";
+import Tooltip from "@leafygreen-ui/tooltip";
+import { Link } from "react-router-dom";
+import { TaskStatusIcon } from "components/TaskStatusIcon";
 import { taskStatusToCopy, mapUmbrellaStatusColors } from "constants/task";
+import { TaskStatus } from "types/task";
+import { pluralize } from "utils/string";
 
 interface Props {
-  status: string;
+  status: TaskStatus;
   count: number;
   onClick?: () => void;
+  statusCounts: { [key: string]: number };
+  href: string;
 }
+
 export const GroupedTaskStatusBadge: React.FC<Props> = ({
   count,
   status,
   onClick = () => undefined,
+  statusCounts,
+  href,
 }) => {
-  let statusDisplayName = taskStatusToCopy[status];
-  if (statusDisplayName.slice(-1) === "s" && count !== 1) {
-    statusDisplayName += "es";
-  } else if (statusDisplayName.slice(-1) === "e" && count !== 1) {
-    statusDisplayName += "s";
-  }
+  const statusDisplayName = pluralize(taskStatusToCopy[status], count);
 
   const { fill, border, text } = mapUmbrellaStatusColors[status];
   return (
-    <BadgeContainer fill={fill} border={border} text={text} onClick={onClick}>
-      <Number>{count}</Number>
-      <Status>{statusDisplayName}</Status>
-    </BadgeContainer>
+    <Tooltip
+      usePortal={false}
+      align="top"
+      justify="middle"
+      popoverZIndex={1}
+      trigger={
+        <Link to={href}>
+          <BadgeContainer
+            fill={fill}
+            border={border}
+            text={text}
+            onClick={onClick}
+          >
+            <Number>{count}</Number>
+            <Status>{statusDisplayName}</Status>
+          </BadgeContainer>
+        </Link>
+      }
+      triggerEvent="hover"
```

Comment:
Perhaps a LeafyGreen bug, but the tooltip placement here isn't looking ideal. Removing `usePortal={false}` appears to fix it, not sure if there's another reason we can't use a portal though!

<img width="162" alt="image" src="https://user-images.githubusercontent.com/9298431/136280013-d12bf30e-eae0-4289-be4d-fb42b851589b.png">